src/helpers/get-error-handler.ts

```diff
@@ -23,7 +23,7 @@ export function getErrorHandler(log: Logger) {
       if (errMessage.includes("pem") || errMessage.includes("json web token")) {
         log.error(
           error,
-          "Your private key (usually a .pem file) is not correct. Go to https://github.com/settings/apps/YOUR_APP and generate a new PEM file. If you're deploying to Now, visit https://probot.github.io/docs/deployment/#now."
+          "Your private key (usually a .pem file) is not correct. Go to https://github.com/settings/apps/YOUR_APP and generate a new PEM file. A wrong `APP_ID` can also cause this issue. If you're deploying to Now, visit https://probot.github.io/docs/deployment/#now."
```

Comment:
How about this

```suggestion
          "Your private key (a .pem file or PRIVATE_KEY environment variable) or APP_ID are incorrect. Go to https://github.com/settings/apps/YOUR_APP, verify that APP_ID is set correctly, and generate a new PEM file if necessary."
```

I removed the note on `now` deployment. It's unrelated, but obsolete.

Can you run `npx jest -u` to update the test snapshots? That should make the tests pass again, too